lib/server/index.js

```diff
@@ -60,9 +62,15 @@ app.use('/graphql', graphqlHTTP(req => ({
 })));
 
 app.use(async (req, res, next) => {
+
+  const metaSettings = {
+    'description': 'description', 'keywords': 'keywords',
+    'copyright':'copyright', 'robots':'robots', 'dctitle': 'DC.title'
+  };
```

Comment:
Check eslint validation. Don't use strings as keys, and for coherence divide each item in a new line:

```js
const metaSettings = {
  description: 'description',
  keywords: 'keywords',
  copyright: 'copyright'
  robots: 'robots',
  dctitle: 'DC.title'
};
```